Makefile

```diff
@@ -0,0 +1,242 @@
+#    offers with one another. The Mediator instance represents a Bisq
+#    contributor who can help resolve any technical problems or disputes
+#    that come up between the two traders.
+#
+# STEP 3: Configure the mediator Bisq node. In order to make and take
+# offers, Alice and Bob will need to have a mediator and a refund agent
+# registered on the network. Follow the instructions below to complete
+# that process:
+#
+#  a) Go to the Account screen in the Mediator instance and press CMD+N
+#  and a popup will appear. Click 'Unlock' and then click 'Register' to
+#  register the instance as a mediator.
+#
+#  b) While still in the Account screen, press CMD+D and follow the same
+#  steps as above to register the instance as a refund agent.
+#
+# When the steps above are complete, your localnet should be up and
+# ready to use. You can now test in isolation all Bisq features and use
+# cases.
+#
+
+
+# Set up everything necessary for deploying your localnet. This is the
+# default target.
+setup: build localnet
+
+clean: clean-build clean-localnet
+
+clean-build:
+	./gradlew clean
+
+clean-localnet:
+	rm -rf localnet
+
+# Build all Bisq binaries and generate the shell scripts used to run
+# them in the targets below
+build:
+	./gradlew build
+
+# Unpack and customize a Bitcoin regtest node and Alice and Bob Bisq
+# nodes that have been preconfigured with a blockchain containing the
+# BSQ genesis transaction
+localnet:
+	# Unpack the old dao-setup.zip and move things around for more concise
+	# and intuitive naming. This is a temporary measure until we clean these
+	# resources up more thoroughly.
+	unzip docs/dao-setup.zip
+	mv dao-setup localnet
+	mv localnet/Bitcoin-regtest localnet/bitcoind
+	mv localnet/bisq-BTC_REGTEST_Alice_dao localnet/alice
+	mv localnet/bisq-BTC_REGTEST_Bob_dao localnet/bob
+	# Remove the preconfigured bitcoin.conf in favor of explicitly
+	# parameterizing the invocation of bitcoind in the target below
+	rm -v localnet/bitcoind/bitcoin.conf
+	# Avoid spurious 'runCommand' errors in the bitcoind log when nc
+	# fails to bind to one of the listed block notification ports
+	echo exit 0 >> localnet/bitcoind/blocknotify
+
+# Deploy a complete localnet by running all required Bitcoin and Bisq
+# nodes, each in their own named screen window. If you are not a screen
+# user, you'll need to run each of the make commands manually in a
+# separate terminal or as a background job.
+#
+# NOTE: You MUST already be attached to a screen session for the
+# following commands to work properly.
+deploy: setup
+	screen -t bitcoin make bitcoind
+	sleep 2    # wait for bitcoind rpc server to start
+	make block # generate a block to ensure Bisq nodes get dao-synced
+	screen -t seednode make seednode
+	screen -t seednode2 make seednode2
+	screen -t alice make alice
+	screen -t bob make bob
+	screen -t mediator make mediator
+
+bitcoind: localnet
+	bitcoind \
+		-regtest \
+		-prune=0 \
+		-txindex=1 \
+		-server \
+		-rpcuser=bisqdao \
```

Comment:
dont we need to add rpcport here?

Reply:
> dont we need to add rpcport here?

No, we're assuming the regtest default value of 18444 everywhere.